src/common/BaseAnnotator.ts

```diff
@@ -58,6 +58,7 @@ export default class BaseAnnotator extends EventEmitter {
                 activeId: fileOptionsValue?.annotations?.activeId ?? null,
             },
             options: {
+                fileCurrentVersionId: file.file_version.id,
```

Comment:
Is the actual file version id needed? Could we just store `isCurrentFileVersion`?

Reply:
This value is already provided to the store two lines down.

Reply:
Disregard my last, I understand the goal now.